README.md

```diff
@@ -13,7 +13,7 @@ Bevy is a refreshingly simple data-driven game engine built in Rust. It is free
 
 ## WARNING
 
-Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes constantly.
+Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes on a roughly quarterly basis.
```

Comment:
The official train release schedule is every 3 months, so I would rephrase it with this in mind. https://bevyengine.org/news/bevy-0-6/#the-train-release-schedule

Personally, I still think the user on hacker news assumed something that was never said. Bevy uses semver and nothing here claims otherwise but I guess it's true that being more accurate doesn't hurt this warning.

Reply:
That link says "approximately once every three months", this says "roughly quarterly". Those would be the same, right?

> Personally, I still think the user on hacker news assumed something that was never said

Agreed on this point

Reply:
Yep, those are the same period of time.

Reply:
> Those would be the same, right?

Correct, what I meant is that I think this should explicitly mention the concept of train release and maybe even link it.